web/src/main/java/org/mskcc/cbio/oncokb/controller/DriveAnnotationParser.java

```diff
@@ -574,17 +576,17 @@ private static void parseTherapeuticImplications(Gene gene, Set<Alteration> alte
 
         // specific evidence
         DrugBo drugBo = ApplicationContextSingleton.getDrugBo();
-        JSONArray drugsArray = implicationObj.has("treatments") ? implicationObj.getJSONArray("treatments") : new JSONArray();
+        JSONArray treatmentsArray = implicationObj.has("treatments") ? implicationObj.getJSONArray("treatments") : new JSONArray();
         int priorityCount = 1;
-        for (int i = 0; i < drugsArray.length(); i++) {
-            JSONObject drugObj = drugsArray.getJSONObject(i);
-            if (!drugObj.has("name") || drugObj.getString("name").trim().isEmpty()) {
+        for (int i = 0; i < treatmentsArray.length(); i++) {
```

Comment:
i < treatmentsArray.length(). Brackets here might be unnecessary.
For array, generally use length. And for String object, use length()?